package-lock.json

```diff
@@ -1,239 +0,0 @@
-{
```

Comment:
This was a stray lock file at the root level, probably added accidentally sometime.